cluster/gce/gci/configure-helper.sh

```diff
@@ -2846,11 +2846,14 @@ function wait-till-apiserver-ready() {
   done
 }
 
-function ensure-bootstrap-kubectl-auth {
+
+function ensure-kubectl-auth {
+  # By default, `kubectl` uses http://localhost:8080
   # Creating an authenticated kubeconfig is only necessary if the insecure port is disabled.
-  if [[ -n "${KUBE_BOOTSTRAP_TOKEN:-}" ]]; then
-    create-kubeconfig "cluster-bootstrap" ${KUBE_BOOTSTRAP_TOKEN}
-    export KUBECONFIG=/etc/srv/kubernetes/cluster-bootstrap/kubeconfig
+  if [[ "${ENABLE_APISERVER_INSECURE_PORT:-false}" != "true" ]]; then
+    KUBE_BOOTSTRAP_TOKEN="$(secure_random 32)"
+    create-kubeconfig "kube-bootstrap" ${KUBE_BOOTSTRAP_TOKEN}
```

Comment:
quote the parameter expansion "${KUBE_BOOTSTRAP_TOKEN}"

Reply:
done